package.json

```diff
@@ -90,7 +90,9 @@
     "react-native-actionsheet": "^2.4.2",
     "react-native-animatable": "^1.3.3",
     "react-native-appearance": "^0.3.4",
+    "react-native-base64": "^0.1.0",
     "react-native-calendars": "^1.403.0",
+    "react-native-camera": "^3.40.0",
```

Comment:
No related @types dep?

Reply:
Not has I see on the documentation:

https://github.com/react-native-camera/react-native-camera